meshPointExample.py

```diff
@@ -20,13 +20,12 @@
 EXPERIMENT_NAME = 'default'
 PROJECT_PATH = 'D:/127 dmso MII O_N pb-02-Airyscan Processing-06_project127 dmso MII O_N pb-02-Airyscan Processing-06_project127 dmso MII O_N pb-02-Airyscan Processing-06_project.syg'
 
-

```

Comment:
I'd prefer to keep this blank line here since the PEP-8 style guide asks for two blanks surrounding top-level functions:

> [Surround top-level function and class definitions with two blank lines.](https://peps.python.org/pep-0008/#blank-lines)